.eslintrc.js

```diff
@@ -1,18 +1,19 @@
 module.exports = {
   env: {
     browser: true,
+    jest: true,
     es2021: true
   },
   extends: ['airbnb-base', 'plugin:prettier/recommended'],
   overrides: [],
-  plugins: ['unused-imports', 'import'],
+  // plugins: ['unused-imports', 'import'],
   parserOptions: {
     ecmaVersion: 'latest',
     sourceType: 'module'
   },
   rules: {
     // Indentation
-    indent: ['error', 2],
+    // indent: ['error', 2],
```

Comment:
add it back, else code indentation will not be checked

Reply:
you might face issue with switch case disable for that
 /* eslint-disable indent */